omod/src/main/webapp/resources/htmlforms/055b-HCTClientCard.xml

```diff
@@ -397,10 +397,15 @@
                     </td>
                 </tr>
                 <tr>
-                    <td colspan="2"><label><span class="required">*</span>Counsellor</label>
-                        <encounterProvider required="true"/>
+                    <td colspan="2"><label>HIV Recency Test Result:</label>
```

Comment:
@echiteri You need to check if the Global Property for Recency is set to true 

See https://github.com/METS-Programme/openmrs-module-aijar/blob/163addfd34a121bb45f59cdbfbaf6aebf5def5c4/omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-SummaryPage.xml#L1058-#L1063